testing/txsim/client.go

```diff
@@ -0,0 +1,283 @@
+package txsim
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/app/encoding"
+	blob "github.com/celestiaorg/celestia-app/x/blob/types"
+	sdkclient "github.com/cosmos/cosmos-sdk/client"
+	auth "github.com/cosmos/cosmos-sdk/x/auth/types"
+	bank "github.com/cosmos/cosmos-sdk/x/bank/types"
+	protogrpc "github.com/gogo/protobuf/grpc"
+	"github.com/tendermint/tendermint/rpc/client/http"
+	coretypes "github.com/tendermint/tendermint/rpc/core/types"
+	"github.com/tendermint/tendermint/types"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+// how often to poll the network for the latest height
+const (
+	DefaultPollTime = 3 * time.Second
+	maxRetries      = 20
+)
```

Comment:
[nit] if more variables are added to the const block, it may not be immediately clear which const this comment refers to

```suggestion
const (
	// DefaultPollTime is how often to poll the network for the latest height
	DefaultPollTime = 3 * time.Second
	maxRetries      = 20
)
```